src/processing/Processor.java

```diff
@@ -0,0 +1,104 @@
+package processing;
+
+import java.util.*;
+import processing.utility.*;
+
+/**
+* This file implements all the function and uses the static functions defined by some other classes.
+* UI will get an object of this class after they call the getProcessor function of ProcessingFactory class.
+*
+* @author Himanshu Jain
+*/
+
+public class Processor implements IDrawErase, IDrawShapes, IOperation, IUndoRedo, IUser {
+	
+	protected Processor() {};
+	
+	@Override
+	public void drawCurve (ArrayList <Pixel> pixels) {
+		return;
+	}
+	
+	@Override
+	public void erase (ArrayList <Position> position) {
+		return;
+	}
+	
+	@Override
+	public void draw_circle(Pixel center, float radius) {
```

Comment:
Please keep consistency in the coding standards. lower camel case is expected for method names.
[Method names convention](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names)